yamllint/rules/key_ordering.py

```diff
@@ -16,8 +16,12 @@
 
 """
 Use this rule to enforce alphabetical ordering of keys in mappings. The sorting
-order uses the Unicode code point number. As a result, the ordering is
-case-sensitive and not accent-friendly (see examples below).
+order uses the Unicode code point number as a default. As a result, the
+ordering is case-sensitive and not accent-friendly (see examples below).
+This can be changed by setting "locale". This is passed to pythons
```

Comment:
For clarity I propose

    This can be changed by setting the ``locale`` option.